src/software/parameter/config/ai.yaml

```diff
@@ -205,7 +205,7 @@ AIConfig:
         type: "double"
         default: 0.18
         description: >-
-            The total width ("sideways" of the robot) of the rectangle starting
+            The total width ('sideways' of the robot) of the rectangle starting
```

Comment:
This string has to get tossed into the boost cli "description" argument

So instead of adding a special case/adding escape characters, its just easier to use a single quote so it doesn't conflict with the string double-quotes